### ISSUE TITLE
Results not same as mentioned in paper

### ISSUE THREAD

**Issue by neermat** [JamesChuanggg/san-torch#1]
Hi,

Thanks for sharing the code. I ran the code with default settings and only got 45% accuracy (in the paper authors have mentioned over 57% accuracy). I noticed that the optim method was set to 'rmsprop' by default, I changed it to 'sgd' since in the paper they have used sgd, and the accuracy dropped to 22%.

I was wondering if you've managed to get accuracy better than this, if yes, could you share your settings, optim method, learning rate, no of iters, etc.

**Comment by badripatro** [comment]
Hi,

Thanks JamesChuanggg for sharing San code in the torch.
I have followed all the steps you have mentioned . I got only validation accuracy   0.2702 as i mentioned below. Could you please inform me, how  to get accuracy mentioned in the paper?

## Here is the training log, which I got after so many iterations,

validation loss:    17.691484603882 accuracy    0.268  
wrote json checkpoint to save/train_vgg_MODEL/checkpoint.json.json  
iter 612600: 0.000002, 0.000041, 0.000189, 238499.336861  
iter 613200: 0.000000, 0.000008, 0.000188, 238785.412024  
iter 613800: 0.000000, 0.000009, 0.000188, 239069.552837  
iter 614400: 0.000001, 0.000012, 0.000188, 239354.925441  
iter 615000: 0.000000, 0.000087, 0.000188, 239638.285724  
iter 615600: 0.000003, 0.000008, 0.000188, 239922.793472  
iter 616200: 0.000000, 0.000011, 0.000188, 240207.835322  
iter 616800: 0.000000, 0.000020, 0.000188, 240491.260781  
iter 617400: 0.000041, 0.000076, 0.000187, 240775.699347  
iter 618000: 0.000001, 0.000094, 0.000187, 241059.249242  
 [=========== 5000/5000 ===============>]  Tot: 4s512ms | Step: 0ms  
validation loss:    17.661752929687 accuracy    0.2702  
wrote json checkpoint to save/train_vgg_MODEL/checkpoint.json.json  
iter 618600: 0.000000, 0.000003, 0.000187, 241348.127413  
iter 619200: 0.000004, 0.000004, 0.000187, 241632.993593  
iter 619800: 0.000000, 0.000238, 0.000187, 241918.548416  
iter 620400: 0.000000, 0.000029, 0.000187, 242202.647449  
iter 621000: 0.000000, 0.000053, 0.000187, 242486.756262  
iter 621600: 0.000000, 0.000090, 0.000186, 242770.933353  
iter 622200: 0.000001, 0.000011, 0.000186, 243054.221052  
iter 622800: 0.000001, 0.000069, 0.000186, 243337.898873  
iter 623400: 0.000000, 0.000018, 0.000186, 243621.912322  
iter 624000: 0.000000, 0.000071, 0.000186, 243905.830473  
 [==================== 5000/5000 =====>]  Tot: 4s471ms | Step: 0ms  
validation loss:    17.861575164795 accuracy    0.2648  
wrote json checkpoint to save/train_vgg_MODEL/checkpoint.json.json  
iter 624600: 0.000000, 0.000126, 0.000186, 244199.602057  
iter 625200: 0.000000, 0.000025, 0.000186, 244484.375617  
iter 625800: 0.000026, 0.000107, 0.000186, 244768.872576  
iter 626400: 0.000001, 0.000022, 0.000185, 245053.777442  
iter 627000: 0.000000, 0.000101, 0.000185, 245338.141863  
iter 627600: 0.000002, 0.000087, 0.000185, 245622.111639  
iter 628200: 0.000000, 0.000047, 0.000185, 245912.373380  
iter 628800: 0.000000, 0.000036, 0.000185, 246197.078349  
iter 629400: 0.000000, 0.000008, 0.000185, 246481.854340  
iter 630000: 0.000001, 0.000040, 0.000185, 246765.680764  
 [========== 5000/5000 =========>]  Tot: 4s406ms | Step: 0ms  

---
